examples/public-server/user.data

```diff
@@ -1 +1,72 @@
-Test test test
+#cloud-config
+
+users:
+- default
+- name: stack
+  sudo: ["ALL=(ALL) NOPASSWD:ALL\nDefaults:stack !requiretty"]
+  shell: /bin/bash
+  ssh_authorized_keys:
+  - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDBoTk3cat4yl1CFTkn75lfWqPWp96gVzpYqpWcLdj3fTUBZC9kFCbkcOkWRSXNB7eWr3ulLEAcyn0byeusT190GQ1rdlSyp6lYX9vLnPYbnfr8JzhX10B8LxSxVxj5JTCqIekBFco+rHO9h3GNF1DCyPIwDfPVl8cpL94g0Id7SVE4wH1GeVIkGoLxgt2R5WNdaxQH3Z4DclvXVwys/FMnkDUC2oTs24oxQMhcinpKaFA12LxWh+vv0+ALoGG6h6m0ka1ELFdHIXULEV5VM4dvMz8GFxvZWGzKHyjiaBsRGhO2takw/E+t/A2cFWhTqCggtK7k0ZEkUXQCbw7W4Q1JainElboCJinPEa7ErLnu0EwfOgIh0IfhyMd1UIAkkd2TAGUmiqpG5QjA/m36m1/CCrOWvyCHftmVOKxICRevzmZVCPSv1DCr3YpYwM7gdPKChACXQ8J4HBcPDNSUVTTHxklUpIqigmpwuZE/ohhs/mYRacPvEPGl+0lbmpuaCMK0e9mmm/0YhUqO9CjMNxShlWOo3hQhOl/kxtMduYmVPpePhE8ZYuaWkpUqVT1FuZ9LZzdWdp74Ero5cSpqHgLWZ7yt7g1JUzyu6xwWeMEQx5vJbD3fC2KBDZUmlica1wLX96dhcXYxjcdxMyGBpeEbScXU5WHJOsIx7DDi9HDu0w==
```

Comment:
We'll need to add the user's key, possibly via kustomize?

Reply:
We can't, unfortunately. This is just an opaque blob to kustomize.

Until we have tooling to generate this stuff I think the simplest solution for this problem specifically is to support keypairs. I opened https://github.com/k-orc/openstack-resource-controller/issues/136 to track adding that.